.nvmrc

```diff
@@ -0,0 +1 @@
+8.12.0
```

Comment:
@kyle-bowden I am not sure if we actually need nodejs for this project, would you be able to see if it builds without it?